ddp-workspace/projects/ddp-atcp/src/app/models/registration-status.ts

```diff
@@ -1,12 +1,12 @@
 export enum RegistrationStatus {
-  NotRegistered = '0',
-  Registered = '1',
-  ConsentedNeedsAssent = '2',
-  Consented = '3',
-  SubmittedPhysicianInfo = '4',
-  SubmittedMedicalHistory = '5',
-  SubmittedGenomeStudyShippingInfo = '6',
-  SubmittedEnrollment = '7',
+  NotRegistered = 'NotRegistered',
+  Registered = 'Registered',
+  ConsentedNeedsAssent = 'ConsentedNeedsAssent',
+  Consented = 'Consented',
+  SubmittedPhysicianInfo = 'SubmittedPhysicianInfo',
+  SubmittedMedicalHistory = 'SubmittedMedicalHistory',
+  SubmittedGenomeStudyShippingInfo = 'SubmittedGenomeStudyShippingInfo',
+  SubmittedEnrollment = 'SubmittedEnrollment',
   Enrolled = '8',
   NotEligible = '9',
   Duplicate = '10',
```

Comment:
@ssettipalli Are we going to map these other status values?

Reply:
Besides, DSM configuration for this "Registration Status" field still has old number values.

Reply:
As per DavidO, they will be updated to those statuses in DSM by study staff manually